csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.Nuget.cs

```diff
@@ -14,6 +16,13 @@ private void RestoreNugetPackages(List<FileInfo> allNonBinaryFiles, IEnumerable<
         {
             try
             {
+                var checkNugetFeedResponsiveness = EnvironmentVariables.GetBoolean(EnvironmentVariableNames.CheckNugetFeedResponsiveness);
```

Comment:
Is this environment variable introduced to let the field team "test" whether the feed checking works before we set it as the default?

Reply:
Yes, that's the idea, but I haven't checked with them yet.